src/isotools/_utils.py

```diff
@@ -179,12 +179,13 @@ def _interval_dist(a, b):
     return max([a[0], b[0]])-min([a[1], b[1]])
 
 
-def events_dist(e1, e2):
+def events_dist(e1, e2, sg):
     '''
```

Comment:
as you need the graph, make it a function of the graph class e.g. def events_dist(self, e1, e2)

Reply:
Done